src/rest.cpp

```diff
@@ -1006,6 +1007,45 @@ static bool rest_blockhash_by_height(const std::any& context, HTTPRequest* req,
     }
 }
 
+static bool rest_broadcast(const std::any& context, HTTPRequest* req, const std::string& str_uri_part)
+{
+    if (!CheckWarmup(req))
+        return false;
+    const std::string body = req->ReadBody();
+    std::string params;
+    const RESTResponseFormat rf = ParseDataFormat(params, str_uri_part);
+    if (params != "") {
+        return RESTERR(req, HTTP_BAD_REQUEST, "Invalid URI format. Expected /rest/broadcast.hex");
+    }
+
+    CMutableTransaction mtx;
+    if (!DecodeHexTx(mtx, body)) {
+        return RESTERR(req, HTTP_BAD_REQUEST, "TX decode failed");
+    }
+
+    switch (rf) {
+    case RESTResponseFormat::HEX: {
+        const CTransactionRef tx(MakeTransactionRef(std::move(mtx)));
+        std::string err_string;
+        NodeContext* node = GetNodeContext(context, req);
+        if (!node) return false;
+
+        const node::TransactionError error = node::BroadcastTransaction(*node, tx, err_string, /*max_tx_fee=*/0, /*relay=*/true, /*wait_callback=*/true);
```

Comment:
I wanted to keep the API easier by passing only the transaction hex in the body, instead of asking for a whole JSON with optional `maxfeerate`/`maxburnamount` (same parameters as sendrawtransaction), and without the ability to set the maxfeerate, having zero (=suppress the check) is better than having the default max fee rate (otherwise you wouldn't be able to broadcast a very high-paying transaction even if you really wanted to).

Additionally, I think most people that would use this API would use it on some external node and not their own, and so they wouldn't delegate the feerate check to the REST interface, but they would do it themselves.

I'm also ok with adding a sentence to the documentation explaining that the maxfeerate is not checked here, although I fear that it might be confusing instead of helpful.

Anyways, I don't really have strong opinions, so I'm totally ok with changing this :)